docs/pages/eas-update/asset-selection.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Asset selection and exclusion
+description: Select which assets should be included in updates.
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { GithubIcon } from '@expo/styleguide-icons';
+
+> **info** Available for SDK 50 and above ([`expo-updates`](/versions/latest/sdk/updates/) >= 0.23.0).
+
+In SDK 49 and earlier, all assets resolved in the Metro bundler are included in every update, and are uploaded to the update server. The new experimental asset selection feature in SDK 50 allows the developer to specify that only certain assets should be included in updates. This has the potential to greatly reduce the number of assets that need to be uploaded to and downloaded from the updates server. This feature will work with the EAS update server, or any custom server that complies with the [`expo-updates` protocol](/technical-specs/expo-updates-1.mdx).
+
+To use this feature, a new property `extra.updates.assetPatternsToBeBundled` should be included in your Expo config. It should define one or more file matching patterns (regular expressions). For example, suppose your `app.json` file has the patterns defined this way:
+
+```json app.json
+  "expo": {
+    /* @hide ... your existing configuration */ /* @end */
+    "extra": {
+      "updates": {
+        "assetPatternsToBeBundled": [
+          "app/images/**/*.png"
+        ]
+      }
+    }
+  }
+```
+
+In this case, all `.png` files in all subdirectories of `app/images` would be included in updates.
+
+Assets that do not match any of the file patterns will still resolve in the Metro bundler, but will not be uploaded to the updates server. If making use of this feature, a developer needs to be certain that assets not included in updates are built into the native build of the app.
+
+If this new property is not in your Expo config, all assets resolved by the bundler will be included in updates (SDK 49 and earlier behavior).
+
+We have created a [complete working demo](https://github.com/expo/UpdatesAPIDemo/blob/main/ASSETSELECTION.md) to allow developers to try out this feature.
```

Comment:
```suggestion
```

Let's remove this since we have the example in "More" section which is the next section so that we can avoid duplicating it. I'm also going suggest to rename the section from "More" to "Example".